plugin/unimpaired.vim

```diff
@@ -125,8 +125,32 @@ function! s:fnameescape(file) abort
   endif
 endfunction
 
-nnoremap <silent> <Plug>unimpairedDirectoryNext     :<C-U>edit <C-R>=<SID>fnameescape(fnamemodify(<SID>FileByOffset(v:count1), ':.'))<CR><CR>
-nnoremap <silent> <Plug>unimpairedDirectoryPrevious :<C-U>edit <C-R>=<SID>fnameescape(fnamemodify(<SID>FileByOffset(-v:count1), ':.'))<CR><CR>
+function! s:PreviousFileEntry(count) abort
+  let window = getwininfo(win_getid())[0]
+
+  if window.quickfix
+    execute 'colder ' . a:count
```

Comment:
This is not ideal, as we are leaking a raised exception when trying to do `:colder` in a quickfix list that does not have any previous entries

```
Error detected while processing function <SNR>57_PreviousFileEntry:
line    4:
E380: At bottom of quickfix stack
error list 1 of 0; 0 errors
```

Is that something we should try / catch to prettify somehow?

Reply:
76e2cc0 seemingly took care of the error message too. Resolving.

Reply:
Yeah that was the sole intent.